Riot/Modules/Common/Recents/Views/RecentTableViewCell.m

```diff
@@ -124,10 +124,10 @@ - (void)render:(MXKCellData *)cellData
             self.roomTitle.font = [UIFont systemFontOfSize:17 weight:UIFontWeightMedium];
         }
 
-        [self.roomAvatar vc_setRoomAvatarImageWith:roomCellData.avatarUrl
-                                            roomId:roomCellData.roomIdentifier
-                                       displayName:roomCellData.roomDisplayname
-                                      mediaManager:roomCellData.mxSession.mediaManager];
+//        [self.roomAvatar vc_setRoomAvatarImageWith:roomCellData.avatarUrl
+//                                            roomId:roomCellData.roomIdentifier
+//                                       displayName:roomCellData.roomDisplayname
+//                                      mediaManager:roomCellData.mxSession.mediaManager];
```

Comment:
i would prefer to remove `RecentTableViewCell` class from the Tchap source (instead of modify the code) , because (I think) this is not used for the moment

Reply:
Removed in latest commits